lib/ec2/host/ec2_client.rb

```diff
@@ -38,7 +39,21 @@ def credentials
         if Config.aws_access_key_id and Config.aws_secret_access_key
           Aws::Credentials.new(Config.aws_access_key_id, Config.aws_secret_access_key)
         else
-          Aws::SharedCredentials.new(profile_name: Config.aws_profile, path: Config.aws_credential_file)
+          config = Config.aws_config
```

Comment:
Prefer `aws_config = Config.aws_config` because `config` in EC2::Host represents EC2::Host::Config.

Reply:
OK. Thanks.